template.hbr

```diff
@@ -275,7 +286,7 @@ players:
                     "font_weight": null
                 },
                 "string.regex": {
-                    "color": "#{{pink}}",
+                    "color": "#{{peach}}",
                     "font_style": null,
                     "font_weight": null
                 },
```

Comment:
This should stay `pink` as per the style guide

Reply:
> Regex should be pink but if it can be more in detail, it should inherit the other colour mappings (e.g. numbers in regex should be peach, lookaheads being blue since they act like a function taking values, etc) - See VSCode's PR on this (https://github.com/catppuccin/vscode/pull/263)

https://github.com/catppuccin/catppuccin/pull/2109#issuecomment-1937378812